adapters/webhdfs-adapter/src/test/java/com/connexta/replication/adapters/webhdfs/WebHdfsClientTest.java

```diff
@@ -109,6 +115,94 @@ public void testStoppedClusterIsNotAvailable() throws Exception {
     adapter.isAvailable();
   }
 
+  @Test
+  public void testResourceFileIsRead() throws URISyntaxException {
+    String testId = "order-66-clones-memo";
+    Date testDate = new Date();
+    String testName = String.format("%s_%s", testId, testDate.getTime());
```

Comment:
✏️  This format shows up a lot in our code. Maybe we could add in a static method for consistency